helios-services/src/test/java/com/spotify/helios/master/DeploymentGroupTest.java

```diff
@@ -139,7 +139,7 @@ public void testUpdateDeploymentGroupHosts() throws Exception {
         .setName(GROUP_NAME)
         .setHostSelectors(ImmutableList.of(HostSelector.parse("role=melmac")))
         .setJobId(job.getId())
-        .setRolloutOptions(RolloutOptions.newBuilder().build())
+        .setRolloutOptions(RolloutOptions.DEFAULT_ROLLOUT_OPTIONS)
```

Comment:
We should add a test where the `RolloutOptions` are partially specified.

Reply:
If I add a unit test for the fallback logic in ZKMM, do we still need this?

Reply:
Are you willing to assert that every codepath that constructs a `DeploymentGroup` does so with a `RolloutOptions` that has all fields filled in?

Reply:
Do you think enforcing this in the `DeploymentGroup` constructor is cleaner than adding a bunch of null checks everywhere we do `DeploymentGroup.getRolloutOptions().getXXX()`?